.gitattributes

```
@@ -1,19 +1,21 @@
 docs/assets/* filter=lfs diff=lfs merge=lfs -text
-disassemblers/ofrak_ghidra/ofrak_ghidra_test/assets/* filter=lfs diff=lfs merge=lfs -text
-disassemblers/ofrak_ghidra/ofrak_ghidra_test/assets/Makefile !filter !diff !merge text
+disassemblers/ofrak_ghidra/tests/assets/* filter=lfs diff=lfs merge=lfs -text
+disassemblers/ofrak_ghidra/tests/assets/Makefile !filter !diff !merge text
+disassemblers/ofrak_angr/tests/assets/* filter=lfs diff=lfs merge=lfs -text
 ofrak_patch_maker/technical_docs/vbcc.pdf filter=lfs diff=lfs merge=lfs -text
-ofrak_core/test_ofrak/components/assets/**/* filter=lfs diff=lfs merge=lfs -text
-ofrak_core/test_ofrak/components/assets/**/Makefile !filter !diff !merge text
-ofrak_core/test_ofrak/components/assets/**/*.md !filter !diff !merge text
-ofrak_core/test_ofrak/components/assets/**/*.c !filter !diff !merge text
-ofrak_core/test_ofrak/components/assets/**/*.sh !filter !diff !merge text
+ofrak_core/tests/components/assets/**/* filter=lfs diff=lfs merge=lfs -text
+ofrak_core/tests/components/assets/**/Makefile !filter !diff !merge text
+ofrak_core/tests/components/assets/**/*.md !filter !diff !merge text
+ofrak_core/tests/components/assets/**/*.c !filter !diff !merge text
+ofrak_core/tests/components/assets/**/*.sh !filter !diff !merge text
 ofrak_tutorial/assets/* filter=lfs diff=lfs merge=lfs -text
 docs/user-guide/gui/assets/* filter=lfs diff=lfs merge=lfs -text
 frontend/public/themes/**/* filter=lfs diff=lfs merge=lfs -text
-disassemblers/ofrak_angr/ofrak_angr_test/assets/* filter=lfs diff=lfs merge=lfs -text
-disassemblers/ofrak_pyghidra/ofrak_pyghidra_test/assets/**/* filter=lfs diff=lfs merge=lfs -text
-disassemblers/ofrak_cached_disassembly/ofrak_cached_disassembly_test/assets/**/* filter=lfs diff=lfs merge=lfs -text
-disassemblers/ofrak_cached_disassembly/ofrak_cached_disassembly_test/assets/Makefile !filter !diff !merge text
-ofrak_core/pytest_ofrak/elf/assets/* filter=lfs diff=lfs merge=lfs -text
-ofrak_core/pytest_ofrak/elf/assets/*.c !filter !diff !merge text
-ofrak_core/pytest_ofrak/elf/assets/Makefile !filter !diff !merge text
+disassemblers/ofrak_angr/tests/assets/* filter=lfs diff=lfs merge=lfs -text
+disassemblers/ofrak_pyghidra/tests/assets/**/* filter=lfs diff=lfs merge=lfs -text
+disassemblers/ofrak_cached_disassembly/tests/assets/**/* filter=lfs diff=lfs merge=lfs -text
+disassemblers/ofrak_cached_disassembly/tests/assets/Makefile !filter !diff !merge text
+pytest_ofrak/src/pytest_ofrak/elf/assets/* filter=lfs diff=lfs merge=lfs -text
+pytest_ofrak/src/pytest_ofrak/elf/assets/*.c !filter !diff !merge text
+pytest_ofrak/src/pytest_ofrak/elf/assets/Makefile filter=lfs diff=lfs merge=lfs -text
```

```suggestion
pytest_ofrak/src/pytest_ofrak/elf/assets/Makefile !filter !diff !merge text
```

Why are we specifically LFS-tracking this Makefile?